### PR TITLE
robbing will not attempt to work on nuker or powerspawn

### PR DESCRIPTION
**Pull request by karlthepagan**
also I'm terrible and didn't document robbing before I forgot it

also this is in 0.7.0

### CODE REVIEW

**Comment by chobobobo** [comment]
This resolves https://github.com/ScreepsOCS/screeps.behaviour-action-pattern/issues/789

**Comment by karlthepagan** [comment]
I had a recent opportunity to rob a power spawn and my robbers sat next to it doing nothing.

It might be nice to generalize this solution by looking for ERR_INVALID_TARGET and banning targets that have that return.

**Comment by chobobobo** [comment]
Well looking at the engine code, it appears powerSpawn is a valid withdraw target for both energy and power, so perhaps there was another reason it was invalid.  A check for ERR_INVALID_TARGET would address this.

**Comment by karlthepagan** [comment]
Ok let's investigate for 7.1

**Comment by chobobobo** [comment]
Ok, so withdraw is valid for a powerSpawn, but I just confirmed that there is a bug in the engine code preventing it from working currently, it always returns ERR_NOT_ENOUGH_RESOURCES
http://support.screeps.com/hc/en-us/community/posts/115002185869-bug-in-src-game-creeps-js-withdrawing-from-power-spawn?page=1#community_comment_115001366669

File: creep.action.robbing.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ action.isValidTarget = function(target){
         return false;
     }
 
+    if (target.structureType === STRUCTURE_NUKER || target.structureType === STRUCTURE_POWER_SPAWN) {
```
Comment (chobobobo) [inline creep.action.robbing.js]: Add a comment here saying why PowerSpawns are disabled so we don't forget to re-enable it later, perhaps with a link to the bug issue, and I'm happy to merge this in

Comment (karlthepagan) [inline creep.action.robbing.js]: +1

**Pull request closed without merging**